### PR TITLE
fix: improve file investigation results

### PR DESCRIPTION
**Pull request by rjsparks**

### CODE REVIEW

File: ietf/doc/templatetags/ietf_filters.py [hunk 1]
```diff
@@ -906,13 +906,17 @@ def mtime(path):
...
+    return path.stat().st_mtime == 0
+
 @register.filter
 def url_for_path(path):
     """Consructs a 'best' URL for web access to the given pathlib Path object.
 
     Assumes that the path is into the Internet-Draft archive or the proceedings.
     """
-    if path.match(f"{settings.AGENDA_PATH}/**/*"):
+    if Path(settings.AGENDA_PATH) in path.parents:
```
Comment (jennifer-richards) [inline ietf/doc/templatetags/ietf_filters.py:919]: These are not exactly synonymous: 
```python
>>> p = Path("/a/b/c")
>>> Path("/a/b/") in p.parents
True
>>> p.match("/a/b/**/*")
False
```

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
Only "Requesting changes" to ask whether the observation in my inline comment is worrying.

File: ietf/doc/templatetags/ietf_filters.py (hunk 1 above)
Comment (rjsparks) [reply ietf/doc/templatetags/ietf_filters.py:919]: Right - this is exactly one of the issues this PR is addressing.

Comment (jennifer-richards) [reply ietf/doc/templatetags/ietf_filters.py:919]: Ah- great!

**Review by jennifer-richards — APPROVED** [review]

**Pull request merged**